ui/lib/kubernetes/addon/routes/fetch-config.js

```diff
@@ -0,0 +1,25 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class KubernetesFetchConfigRoute extends Route {
```

Comment:
The overview, configuration and roles routes all need to display a call to action if the user has not yet configured the engine so I thought I would create this route and extend it where needed.

Reply:
This might be helpful to add as a comment!

Reply:
Ya good call let me go ahead and add that!